### PR TITLE
62 add aif conversion script to pipeline

### LINKED ISSUE #62: Add AIF conversion script to pipeline
https://github.com/isi-vista/aida-tools/pull/384

### PR DESCRIPTION
**Pull request by elizlee**
Closes #62 

Todos: #123

### CODE REVIEW

File: cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+                + make_xml_safe(cdse_system + source + "/claim_id/" + claim_id)
+                + ">"
+            )
+            af.write(claim_name + " a aida:Claim ;\n")
+            af.write('\taida:sourceDocument "' + str(data["doc_id"]) + '"^^xsd:string ;\n')
+            claim_id = make_xml_safe(str(data["claim_id"]))
+            af.write('\taida:claimId "' + claim_id + '"^^xsd:string ;\n')
+            # Not supported: queryId
+            #      af.write('\taida:queryId "None" ;\n')
+            # Not supported: importance
```
Comment (joecummings) [inline cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:239]: You can remove these.

File: cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+            claimer_name = "None"
+            claimer_entity_name = "None"
+            claimer_justification_name = "None"
+
+            if has_claimer:
+                claimer_name, claimer_entity_name, claimer_justification_name = (
+                    get_name_data(data, "claimer", source, claim_id)
+                )
+
+            # Not supported optional: claimerAffiliation
```
Comment (joecummings) [inline cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:321]: We don't add the epistemic status, etc for the claim and probably never will so I don't think it's a TODO.

File: cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+            if has_claim_date_time:
+                # TODO: no claim_date_time values have been sent, they will need to be converted to LDCTime
+                af.write("\taida:claimDateTime aida:TimeTBD ;\n")
+
+            has_claim_location = data["claim_location_qnode"] is not None
+            claim_location = "None"
+            if has_claim_location:
+                claim_location = make_xml_safe(data["claim_location_qnode"])
+                af.write("\taida:claimLocation aida:" + claim_location + " ;\n")
+            # Not supported: associatedKEs, identicalClaims, relatedClaims, supportingClaims, refutingClaims
```
Comment (joecummings) [inline cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:339]: Aren't we supporting `associatedKEs` now?

File: cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py [hunk 4]
```diff
@@ -0,0 +1,521 @@
...
+            if has_x_variable:
+                write_qnode_data(
+                    data,
+                    "x_variable",
+                    x_variable_name,
+                    x_variable_entity_name,
+                    x_variable_justification_name
+                )
+
+            # <http://www.isi.edu/gaia/entities/uiuc/L0C04959A/L0C04959A-author> a aida:Entity ;
```
Comment (joecummings) [inline cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:386]: This was an old "Dan comment", correct? As such, I can be removed.

File: wikidata_linker/linker.py [hunk 5]
```diff
@@ -1,10 +1,9 @@
 """Module of class model for RoBERTa classifier used for Wikidata linking classification."""
 from typing import Any, Optional, Sequence
 
+# from torch.utils.data import DataLoader
```
Comment (joecummings) [inline wikidata_linker/linker.py:4]: Why is this here?

Comment (elizlee) [reply wikidata_linker/linker.py:4]: I don't remember adding it, but I'll remove it

File: cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py [hunk 6]
```diff
@@ -0,0 +1,521 @@
+"""Convert the claim data from the output json into AIF files."""
```
Comment (joecummings) [inline cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:1]: Can you possibly split the large function into descriptive subfunctions?

File: cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py (hunk 4 above)
Comment (elizlee) [reply cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:386]: Yeah, a lot of these are Dan comments...

**Review by joecummings — CHANGES_REQUESTED** [review]
Looks pretty good, just a few comments.

**Review by joecummings — APPROVED** [review]
lgtm
